woocommerce/woocommerce
# prevent line breaks in plugin author names

### All Submissions:






### Changes proposed in this Pull Request:
preventing unintended line breaks in the plain text version of the System Status Report when a plugin lists multiple authors separated by comma, such as:
```
Some plugin: by Some Company
LLC – 1.0
```
instead of:
```
Some plugin: by Some Company, LLC – 1.0
```

Or:

```
Another plugin: by John
Jane
Jack
Josh – 2.0
```
instead of:
```
Another plugin: by John, Jane, Jack, Josh – 2.0
```



Closes # .

### How to test the changes in this Pull Request:

1. Install a plugin with multiple authors (for example: [UpdraftPlus WordPress Backup Plugin](https://wordpress.org/plugins/updraftplus/), which currently lists "UpdraftPlus.Com, DavidAnderson")
2. Go to WooCommerce > Status
3. Click "Get system report" and observe that the plugin authors are no longer separated by newlines.

### Other information:




### Changelog entry

> Fix: author list formatting in plain text system status report

### FOR PR REVIEWER ONLY:
